app/services/recommended_challenge.py

```diff
@@ -0,0 +1,93 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+from uuid import UUID
+from decimal import Decimal
+from typing import List, Optional
+from fastapi import HTTPException
```

Comment:
아래 코멘트 내용들 반영하려면 이게 필요하니 추가해주세용

```suggestion
from fastapi import HTTPException, status
```